src/chain/validators.ts

```diff
@@ -10,6 +10,7 @@ export interface Validators<Return> {
   // custom validators
   custom(validator: CustomValidator): Return;
   exists(options?: { checkFalsy?: boolean; checkNull?: boolean }): Return;
+  if(conditionFn: ConditionFn): Return;
```

Comment:
The same as above. `if` is not a validator, but a context handler, so is defined in that file.